Yes, I'm setting the CFLAGS makefile variable to $CXXFLAGS. This build
system likes to pretend that it's using a C compiler, for some reason.

Index: configure
===================================================================
--- configure.orig
+++ configure
@@ -124,8 +124,8 @@ if ($debug_makefile) {
   print OUT "CFLAGS=-g -Wall\n";
   print OUT "LDFLAGS=-g -Wall\n";
 } else {
-  print OUT "CFLAGS=-O3\n";
-  print OUT "LDFLAGS=-O3\n";
+  print OUT "CFLAGS=$ENV{'CXXFLAGS'}\n";
+  print OUT "LDFLAGS=$ENV{'LDFLAGS'}\n";
 }
 
 close OUT;
